frontend/public/components/catalog/catalog-item-icon.tsx

```diff
@@ -130,6 +131,7 @@ const logos = new Map()
   .set('icon-go-gopher', goLangImg)
   .set('icon-golang', goLangImg)
   .set('icon-grails', grailsImg)
+  .set('icon-gitops', gitopsImg)
```

Comment:
No need to add gitops icon to catalog icons. Import directly from `dev-console/src/images`